scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJobFactory.java

```diff
@@ -385,8 +385,6 @@ private static void setJobCommonProperties(Job job, InternalJob jobToSet)
             throws IllegalArgumentException, IllegalAccessException {
         autoCopyfields(CommonAttribute.class, job, jobToSet);
         autoCopyfields(Job.class, job, jobToSet);
-        jobToSet.setVariables(job.getVariables());
-        jobToSet.setGenericInformation(job.getGenericInformation());
```

Comment:
Can you clarify why we have to remove this ?

Reply:
Indeed it is here by mistake.